ppa/archive/views.py

```diff
@@ -52,8 +51,8 @@ class DigitizedWorkListView(AjaxTemplateMixin, ListView):
     # search title query field syntax
     search_title_query = '{!type=edismax qf=$search_title_qf pf=$search_title_pf v=$title_query}'
 
-
     def get_queryset(self, **kwargs):
+        print("*** get queryset")
```

Comment:
should this still be in here?

Reply:
Ha, nope! I was testing whether the view is still getting called (it does) and had another thought about how we might avoid generating the whole response but decided to wait on that and forgot to take this back out. I'll clean it up.